packages/mdc-list/index.js

```diff
@@ -89,13 +94,47 @@ export class MDCList extends MDCComponent {
     this.foundation_.setWrapFocus(value);
   }
 
+  /** @param {boolean} value */
+  set singleSelection(value) {
+    if (value) {
+      this.root_.addEventListener('click', this.handleClick_);
+    } else {
+      this.root_.removeEventListener('click', this.handleClick_);
+    }
+
+    this.foundation_.setSingleSelection(value);
+    const selectedElement = this.root_.querySelector('.mdc-list-item[aria-selected="true"]');
```

Comment:
Could you provide some explanation about why this uses the aria-selected attribute instead of the `mdc-list-item--selected` class?